include/bluetooth/services/ots.h

```diff
@@ -496,20 +524,23 @@ struct bt_ots_cb {
 	 *  object. This callback is also triggered when the server
 	 *  creates a new object with bt_ots_obj_add() API.
 	 *
-	 *  @param ots  OTS instance.
-	 *  @param conn The connection that is requesting object creation or
-	 *              NULL if object is created by the following function:
-	 *              bt_ots_obj_add().
-	 *  @param id   Object ID.
-	 *  @param init Object initialization metadata.
+	 *  @param ots           OTS instance.
+	 *  @param conn          The connection that is requesting object creation or
+	 *                       NULL if object is created by the following function:
+	 *                       bt_ots_obj_add().
+	 *  @param id            Object ID.
+	 *  @param add_param     IN: Object creation requested parameters.
+	 *  @param created_param OUT: Created object parameters.
```

Comment:
```suggestion
	 *  @param add_param     Object creation requested parameters.
	 *  @param created_param Created object parameters that shall be filled by the receiver of this callback.
```

Not sure `OUT` is used the same way as we normally do for `OUT` parameters since it is a struct that needs to be initialized.